modules/migration-network-traffic-redirection-strategies.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+// * migration_toolkit_for_containers/network-considerations-3-4.adoc
+
+[id="migration-network-traffic-redirection-strategies_{context}"]
+= Network traffic redirection strategies
+
+After a successful migration, you must redirect network traffic of your stateless applications from the source cluster to the target cluster.
+
+The methods for redirecting network traffic are based on the following assumptions:
+
+* The application pods are running on both the the source and target clusters.
+* Each application has a route that contains the source cluster hostname.
+* The route with the source cluster hostname contains a CA certificate.
+* If you require secure connections through the source cluster's DNS domain, the target router default certificate includes the source cluster wildcard `Subject Alternative Name`.
+
+Consider the following methods and select the one that meets your objectives.
```

Comment:
Same nit as line 11--my sugg is "Consider the following strategies..." so that all the terminology is in alignment.